modules/MarketAnalysis.py

```diff
@@ -106,9 +174,17 @@ def get_rate_suggestion(self, cur, rates=None, method='percentile'):
             if len(rates) == 0:
                 return 0
             if method == 'percentile':
+                # rates is a tuple with the first entry being unixtime
+                rates = [x[1] for x in rates]
                 return self.get_percentile(rates, self.lending_style)
             elif method == 'golden_cross':
-                raise ValueError("{0} not yet implmented")
+                if len(rates) < 1800:
```

Comment:
In this case, consider a fallback to percentile?
Probably doesn't matter either way.